analysis/notebooks/README.md

```diff
@@ -1,9 +1,6 @@
-# Example Jupyter / Google Colab Notebooks
+# Example Google Colab / Jupyter Notebooks
```

Comment:
Maybe we should just put this document in the real docs? under https://google.github.io/fuzzbench/reference/report/ maybe?

Reply:
Added a subsection in custom_analysis_and_reports.md